workalendar/africa/kenya.py

```diff
@@ -0,0 +1,70 @@
+from copy import copy
+from datetime import timedelta, date
+
+from ..core import WesternCalendar, IslamicMixin, ChristianMixin
+from ..registry_tools import iso_register
+
+SUN = 6
+
+@iso_register('KE')
+class Kenya(WesternCalendar, IslamicMixin, ChristianMixin):
+    "Kenya"
+    include_good_friday = True
+    include_easter_monday = True
+    # Islamic holidays
+    include_eid_al_fitr = True
+    include_day_of_sacrifice = True
+    shift_sunday_holidays = True
+
+    FIXED_HOLIDAYS = WesternCalendar.FIXED_HOLIDAYS + (
+        (5, 1, "Labour Day"),
+        (6, 1, "Madaraka Day"),
+        (10, 20, "Mashujaa Day"),
+        (12, 12, "Jamhuri Day"),
+        (12, 31, "New Years Eve"),
+    )
+
+    def get_fixed_holidays(self, year):
+        days = super().get_fixed_holidays(year)
+
+        if year >= 2020:
+            days.append((date(year, 2, 11), 'Moi Memorial Day'))
+
+        # Moi Day renamed
+        huduma_day_label = "Moi Day"
+        if year >= 2020:
+            huduma_day_label = "Huduma Day"
+        days.append((date(year, 10, 10), huduma_day_label))
+
+        # Boxing day renamed
+        boxing_day_label = "Boxing Day"
+        if year >= 2020:
+            boxing_day_label = "Utamaduni Day"
+        days.append((date(year, 12, 26), boxing_day_label))
+
+        return days
+
+    def get_shifted_holidays(self, dates):
```

Comment:
Added these next two methods to shift Sundays to Mondays, but I've excluded the Islamic days from being shifted.

Reply:
excellent!